editor/src/clj/editor/properties_view.clj

```diff
@@ -594,6 +612,7 @@
                                   (let [c (.getValue color-picker)]
                                     (set-color-value! property-fn ignore-alpha c)
                                     (ui/user-data! (ui/main-scene) ::ui/refresh-requested? true))))
+    (.setOnHidden color-picker (fn [_] (save-colors! color-picker prefs)))
```

Comment:
We also need to save the list when we remove a color, so calling `save-colors!` this on `ui/on-action!` won't suffice.

Reply:
Since ColorPicker's `getCustomColors` returns an observable list, I think it might be better to add a `ChangeListener` to it to save them.